.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,9 +3,36 @@ name: "\U0001F41B Bug Report"
 about: Create a bug report to help us improve DVC
 ---
 
-## Bug Report
+# Bug Report
 
-### Please provide information about your setup
+## Issue name
+
+Issue names must follow the pattern `command: description` where the command is the dvc command that you are trying to run. The description should describe the consequence of the bug. Example: `repro: doesn't detect input changes`
+
+Delete this section before publish
```

Comment:
This kind of thing can be done with comments, so the user will see the message when they open the bug report, but don't have to explicitly remove all of the instructions every time they file a report.

the template would just look something like
```
# Bug Report

<!--
Issue names should follow the pattern ...
Example: `repro: doesn't detect input changes`
-->

...
```

Reply:
This is a good idea! I basically added this to information blocks so they do not appear by default.